pmd-java/src/main/java/net/sourceforge/pmd/lang/java/symbols/internal/ast/AstClassSym.java

```diff
@@ -116,6 +121,14 @@
                 }
             }
         }
+        
+        final NodeStream<ASTAnnotation> annotStream = node.getDeclaredAnnotations();
+        if (annotStream.nonEmpty()) {
+            myAnnotations = new ArrayList<>();
+            annotStream.forEach(anode -> myAnnotations.add(new AstSymbolicAnnot(anode)));
+        } else {
+            myAnnotations = Collections.emptyList();
+        }
```

Comment:
```suggestion
            this.declaredAnnotations = Collections.unmodifiableList(annotStream.toList(ASTSymbolicAnnot::new));
```

The list is not touched anywhere else. So I don't think you need `myAnnotations`